cedar-policy/CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- The `ValidationResult` returned from `Validator::validate` now has a static
+  lifetime, allowing it to be used in more contexts. The lifetime parameter
+  will be removed in a future major version. 
```

Comment:
If we're going to say this, we should make an issue to remember to do it

Reply:
sure